tdrs-frontend/cypress/e2e/common-steps/common-steps.js

```diff
@@ -0,0 +1,18 @@
+import { When } from '@badeball/cypress-cucumber-preprocessor'
+
+When('The user visits the home page', () => {
+  cy.visit('/')
+  cy.contains('Sign into TANF Data Portal', { timeout: 30000 })
+})
+
+When('The new user logs in', () => {
+  cy.login('new-cypress@teamraft.com')
+})
+
+When('The admin logs in', () => {
+  cy.adminLogin('cypress-admin@teamraft.com')
+})
+
+When('The user logs in', () => {
```

Comment:
these are great common steps. if you wanted to parameterize the admin requests to update the account status, you could add them here with something like

`When('the admin sets approval status of {string} to {string}', (username, status) => {})`

used like this
`When the admin sets the approval status of "new-cypress@teamraft.com" to "Approved"`

rough example but hopefully that gives you an idea. https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/docs/cucumber-basics.md